src/components/manage/Widgets/GeolocationWidget.jsx

```diff
@@ -12,62 +16,74 @@ import {
 } from '@plone/volto/components/manage/Widgets/SelectStyling';
 
 const messages = defineMessages({
-  Coverage: {
-    id: 'Geometric coverage',
-    defaultMessage: 'Geometric coverage',
+  coverage: {
+    id: 'Geographic coverage',
+    defaultMessage: 'Geographic coverage',
   },
 });
+const groupedOptions = [
+  {
+    label: 'Biogeographical region',
+    options: biogeographicalData,
+  },
+  {
+    label: 'Countries group',
+    options: eeaCountries,
+  },
+];
+const Group = (props) => <components.Group {...props} />;
 
 const GeolocationWidget = (props) => {
   const { data, block, onChangeBlock, intl } = props;
   const [selectedOption, setOption] = useState(null);
+
   React.useEffect(() => {
     onChangeBlock(block, {
       ...data,
-      geographic: selectedOption
-        ? [...data.geographic, selectedOption]
-        : data.geographic,
+      geolocation: selectedOption ? [...selectedOption] : data.geolocation,
```

Comment:
eslint is complaining:

```
React Hook React.useEffect has missing dependencies: 'block', 'data', and 'onChangeBlock'. Either include them or remove the dependency array.
```

Reply:
Confused with this warning, Can't add block,data because they are causing additional calls. I disabled eslint for this line.